clients/admin-ui/src/theme/tailwind.css

```diff
@@ -1,3 +1 @@
-@tailwind base;
```

Comment:
Were the base styles conflicting with anything? From what I could find, looks like a "normalize" stylesheet, and consistency across browsers. I am wondering if it might be useful or if could conflict with some other normalize we might have.
I think this is the stylesheet it imports: https://unpkg.com/tailwindcss@3.4.14/src/css/preflight.css

Reply:
yeah, both Ant and Chakra are already including a normalize, so this was unnecessarily redundant and had potential to mess with Ant. Once we get rid of Chakra we'll be able to better solve this with `@layer`, but right now that's causing too many conflicts. see https://ant.design/docs/react/compatible-style#compatible-with-third-party-style-libraries